support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -79,11 +94,18 @@ object DigitalPackValidation {
     import createSupportWorkersRequest.product._
     import createSupportWorkersRequest.billingAddress._
 
+    def isCorporateSub(request: CreateSupportWorkersRequest) =
+      request.paymentFields == FreeProduct //TODO: also need a valid code, this is coming in part 2
```

Comment:
got it, we could also check product is digitalpack (or read from dynamo and match it up if we want to be more flexible again in part 2)

Reply:
oh sorry, did't notice this is specifically the digital pack validator! ignore